examples/streaming_transcribe_from_local_file/streaming_transcribe_from_local_file.js

```diff
@@ -0,0 +1,47 @@
+const revai = require('revai-node-sdk');
+const fs = require('fs');
+const token = require('../config/config.json').access_token;
+
+/* Initialize your client with your audio configuration and access token */
```

Comment:
I wonder if it will be confusing for customers to stream non-raw audio file from this example, i.e they have to creat a new client with different `AudioConfig` parameters which is not obvious how to do so from this example. 

We have both local and media url async examples, should we add a non-raw (like a .wav) and stream? Or do you think that's overkill @kbridbur

Reply:
I think it's a bit overkill. We should link the streaming docs in the readme though